.github/workflows/build-test-prValidation.yml

```diff
@@ -49,7 +49,6 @@ jobs:
     strategy:
       matrix: # *DO NOT EDIT* the following line is generated by running ./scripts/gh-generate-projects-matrix.sh from the root folder
         project: [akka-actor,akka-actor-testkit-typed,akka-actor-typed,akka-cluster,akka-cluster-metrics,akka-cluster-sharding,akka-cluster-sharding-typed,akka-cluster-tools,akka-cluster-typed,akka-coordination,akka-discovery,akka-distributed-data,akka-serialization-jackson,akka-multi-node-testkit,akka-osgi,akka-persistence,akka-persistence-query,akka-persistence-typed,akka-persistence-testkit,akka-protobuf,akka-protobuf-v3,akka-pki,akka-remote,akka-slf4j,akka-stream,akka-stream-testkit,akka-stream-typed,akka-testkit,akka-actor-tests,akka-actor-typed-tests,akka-docs,akka-bill-of-materials,akka-persistence-shared,akka-persistence-tck,akka-persistence-typed-tests,akka-remote-tests,akka-stream-tests,akka-stream-tests-tck]
-      fail-fast: true    
```

Comment:
Even after removing `fail-fast: true` when a single job in the matrix fails, all jobs in the matrix are aborted. I think `fail-fast` doesn't mean what I think it means.